ISL.ReIdentification.Core.Tests.Unit/Services/Foundations/UserAccesses/UserAccessesTests.cs

```diff
@@ -44,6 +44,13 @@ private static UserAccess CreateRandomUserAccess() =>
         private static UserAccess CreateRandomUserAccess(DateTimeOffset dateTimeOffset) =>
             CreateUserAccessesFiller(dateTimeOffset).Create();
 
+        private static IQueryable<UserAccess> CreateRandomUserAccesses()
+        {
+            return CreateUserAccessesFiller(GetRandomDateTimeOffset())
+                .Create(GetRandomNumber())
+                .AsQueryable();
+        }
```

Comment:
I would suggest to always use a list here so that you would not have to deal with deferred execution issues


```suggestion
        private static List<UserAccess> CreateRandomUserAccesses()
        {
            return CreateUserAccessesFiller(GetRandomDateTimeOffset())
                .Create(GetRandomNumber())
                .ToList();
        }
```